### PR TITLE
feat (cli, metrics): be able to see at a glance which metrics are from which sources - #PLAT-774

### PR DESCRIPTION
**Pull request by gvelez17**
## Description

Since our prometheus will scrape metrics from multiple sources it is desirable to namespace the sources, both for context of where the metrics are coming from and to prevent stepping on each other. 

This PR is to accept an optional parameter for caller name and prepend it to metric names 

## How Has This Been Tested?

Describe the tests that you ran to verify your changes. Provide instructions for reproduction.

- [ x] tests pass, changes are exercised

## PR checklist

Before submitting this PR, please make sure:

- [x] I have tagged the relevant reviewers and interested parties
- [ ] I have updated the READMEs of affected packages
- [ ] I have made corresponding changes to the documentation

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

(will be documenting once we have approved and tested the metrics, want to finalize the methods first)

### CODE REVIEW

**Review by stbrody — APPROVED** [review]
LGTM except the commit message should include the package names that are updated, and should not reference linear tickets as those are private.  So: `feat(cli,metrics): be able to see at a glance which metrics are from which sources`

**Comment by gvelez17** [comment]
hey @stbrody thx for review, I forgot actually to increment the package version it depends on - since this both updates the metrics package and the code depending on it, is there any order we have to do when merging & deploying?  Should we do the new npm package release first?

File: packages/metrics/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ceramicnetwork/metrics",
-  "version": "0.0.3-rc.1",
```
Comment (stbrody) [inline packages/metrics/package.json:3]: this should probably be reverted also

Comment (gvelez17) [reply packages/metrics/package.json]: yep done

File: packages/metrics/src/metrics-setup.ts [hunk 2]
```diff
@@ -66,7 +72,7 @@ class _Metrics {
     }
     // Create this counter if we have not already
     if (!(name in this.counters)) {
-      this.counters[name] = this.meter.createCounter(name)
+      this.counters[name] = this.meter.createCounter('${caller}:${name}')
```
Comment (stbrody) [inline packages/metrics/src/metrics-setup.ts:75]: this.caller?

File: packages/metrics/src/metrics-setup.ts [hunk 3]
```diff
@@ -79,7 +85,7 @@ class _Metrics {
     }
     // Create this Histogram if we have not already
     if (!(name in this.histograms)) {
-      this.histograms[name] = this.meter.createHistogram(name)
+      this.histograms[name] = this.meter.createHistogram('${caller}:${name}')
```
Comment (stbrody) [inline packages/metrics/src/metrics-setup.ts:88]: this.caller?

File: packages/metrics/src/metrics-setup.ts (hunk 2 above)
Comment (gvelez17) [reply packages/metrics/src/metrics-setup.ts]: ah!  yes. fixed

**Review by stbrody — APPROVED** [review]
lgtm

**Pull request merged**